tests/PySys/apt_plugin/apt_update_list/run.py

```diff
@@ -0,0 +1,53 @@
+from environment_apt_plugin import AptPlugin
+import sys
+import os
+import subprocess
+import time
+
+sys.path.append("apt_plugin")
+
+"""
+Validate apt plugin update-list command use case
+
+When we install/remove multiple packages at once with update-list command
+Then these packages are installed/removed together
+
+Issue:
+whenever there is a parameter --version the output will be "apt-install 0.1.0"
+
+sudo /etc/tedge/sm-plugins/apt install rolldice 1.16-1+b3 --version
+apt-install 0.1.0
+"""
+
+
+class AptPluginUpdateList(AptPlugin):
+
+    update_list = 'update-list'
+
+    def setup(self):
+        super().setup()
+
+        self.package1 = "rolldice"
+        self.package2 = "asciijump"
+        self.package3 = "moon-buggy"
+
+        self.apt_remove(self.package1)
+        self.apt_install(self.package2)
+        self.apt_remove(self.package3)
+
+        self.assert_isinstalled(self.package1, False)
+        self.addCleanupFunction(self.cleanup_prepare)
+
+    def execute(self):
+        os.system('{} {} {} < {}'.format(self.sudo,
+                  self.apt_plugin, self.update_list, self.input + "/update_list_input"))
```

Comment:
Use f string, the code should get a little bit cleaner:
```suggestion
        input_list_path = f'{self.input}/update_list_input'
        os.system(f'{self.sudo} {self.sudo} {self.update_list} < {input_list_path }')
```